www/src/graph/payments.graphql

```diff
@@ -53,6 +71,9 @@ fragment Invoice on Invoice {
   createdAt
   hostedInvoiceUrl
   lines { ...InvoiceItem }
+  paymentIntent {
```

Comment:
This shouldn't be on all invoice structs, since it won't be sideloaded on listing invoices for an account for example